implementations/rust/ockam/ockam_command/src/lib.rs

```diff
@@ -446,3 +448,62 @@ pub(crate) fn replace_hyphen_with_stdin(s: String) -> String {
         s
     }
 }
+
+const EXIT_OK: i32 = 0;
+const EXIT_USAGE: i32 = 2;
```

Comment:
I basically like the suggestion. However, those are copied from `clap::Error::exit` which uses the same exit code for `USAGE` as prominent UNIX commands such as `ls` and the like. Using those constants would be a change in behavior which I feel is outside the scope of this submission.

Reply:
Just to give you some more context, the error codes we use are the ones defined at https://man.freebsd.org/cgi/man.cgi?query=sysexits&apropos=0&sektion=0&manpath=FreeBSD+4.3-RELEASE&format=html

We'll review it again in case we should change them.

Reply:
I already had a previous state return those, but then just ended up using `clap::Error::exit` again for simplicity.

Since you mentioned elsewhere that you see a lot of novice contributors, probably also people who are new to Rust (I include myself in that group), and there's quite some setup to be done in total to get on board, the remapping the of these exit codes could be a really nice and easy follow-up task to file. Basically:

```
clap_error.print();

// from `clap_builder::util::safe_exit` (not needed if we always unwind on exit):
let _ = std::io::stdout().lock().flush();
let _ = std::io::stderr().lock().flush();

let is_usage_error = clap_error.use_stderr();
// might as well replicate(private) `clap::Error::stream()` making it more obvious why

process::exit(if is_usage_error { USAGE } else { OK });
```